test/performance/scheduler/minimalkueue/main.go

```diff
@@ -66,20 +66,24 @@ func init() {
 }
 
 func main() {
-	os.Exit(mainWithExitCode())
+	initFlags()
+	flag.Parse()
+	os.Exit(run())
 }
 
-func mainWithExitCode() int {
-	opts := zap.Options{
-		TimeEncoder: zapcore.RFC3339NanoTimeEncoder,
-		ZapOpts:     []zaplog.Option{zaplog.AddCaller()},
-		Development: true,
-		Level:       zaplog.NewAtomicLevelAt(zapcore.ErrorLevel),
-	}
-	opts.BindFlags(flag.CommandLine)
-	flag.Parse()
-	log := zap.New(zap.UseFlagOptions(&opts))
+var logOptions = zap.Options{
+	TimeEncoder: zapcore.RFC3339NanoTimeEncoder,
+	ZapOpts:     []zaplog.Option{zaplog.AddCaller()},
+	Development: true,
+	Level:       zaplog.NewAtomicLevelAt(zapcore.ErrorLevel),
+}
+
+func initFlags() {
+	logOptions.BindFlags(flag.CommandLine)
+}
 
+func run() int {
```

Comment:
Hm, I would rather return error here also. Then it will allow us to even unit test this in the future, wdyt?